serialbox/install.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -x
+set -e
+
+repo="https://github.com/ai2cm/serialbox"
+workdir=`pwd`
+build_dir=${workdir}/build
+
+machine=$1
```

Comment:
I think you can skip the `machine` part, no?

Reply:
done